include/mxnet/lib_api.h

```diff
@@ -437,10 +438,12 @@ class OpResource {
  public:
   OpResource(xpu_malloc_t cpu_malloc_fp, void* cpu_alloc_fp,
              xpu_malloc_t gpu_malloc_fp, void* gpu_alloc_fp, void* stream,
-             sparse_malloc_t sparse_malloc_fp, void* sparse_alloc_fp)
+             sparse_malloc_t sparse_malloc_fp, void* sparse_alloc_fp,
+             void* cpu_states, void* gpu_states)
```

Comment:
can we call these cpu_rand_states and gpu_rand_states so that its clear that these are for the RNG?